webapp/components/LocaleSwitch/LocaleSwitch.vue

```diff
@@ -64,15 +66,44 @@ export default {
       })
       return routes
     },
+    ...mapGetters({
+      currentUser: 'auth/user',
+    }),
   },
   methods: {
     changeLanguage(locale, toggleMenu) {
       this.$i18n.set(locale)
+      this.updateUserLocale()
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-24T06:44:03Z" title="Thursday, October 24th 2019, 8:44:03 am +02:00">Oct 24, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -64,15 +66,44 @@ export default {
       })
       return routes
     },
+    ...mapGetters({
+      currentUser: 'auth/user',
+    }),
   },
   methods: {
     changeLanguage(locale, toggleMenu) {
       this.$i18n.set(locale)
+      this.change()
```


I don't love this naming convention @ogerly, it doesn't tell me what the function does
shouldn't it be something like `updateUserLocale`?

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-10-24T06:50:35Z" title="Thursday, October 24th 2019, 8:50:35 am +02:00">Oct 24, 2019</time>_

---


yeah, you're right. KISS does not always make sense. ;) I change it.